docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -35,6 +37,35 @@ As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`.
 $ sudo yum install -y chrony
 ```
 
+### PHC (PTP Hardware Clock)
+
+**Supported Platforms:** AWS
+
+{{<tags/feature/tp>}} PTP Hardware Clock (PHC) refers to a physical clock device that supports the [Precision Time Protocol (PTP)](https://en.wikipedia.org/wiki/Precision_Time_Protocol), which is a network protocol used for highly accurate time synchronization.
+
+To check if your AWS instance supports PHC, see [AWS PTP Hardware Clock](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock).
+
+Configure PTP using the `configure_ptp.sh` script in the bin directory of your YugabyteDB home directory as follows:
```

Comment:
Are we configuring PTP or PHC? Is there a distinction here? They appear to be separate, but the section is called PHC not PTP

Reply:
> Are we configuring PTP or PHC?
Good question. We are doing both.

Basically the steps involve
1. Installing a new network driver in the kernel with PHC enabled.
2. Changing chrony to use PHC as a time source and PTP for time sync.

To clarify,
1. time source represents a physical clock (hardware clock) that is giving us timestamp readings (PHC)
2. time sync is making sure our clock is reading the same values as the rest of the world (PTP)

> Is there a distinction here?
Yes.

PHC is a hardware clock.
1. The device must support it (the link indicates which hardware supports this).
2. The kernel must support it (we compile the network driver with PHC support and install it).

PTP is a network protocol used for time synchronization.
PTP decides what messages goes over the network and how the hardware clock (PHC) is adjusted. PTP requires PHC to be available.

Both are configured using the configure_ptp.sh script.

> They appear to be separate, but the section is called PHC not PTP

I've changed the section a little bit to clarify things. Let me know if this is better.